clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/PersistenceDAO.scala

```diff
@@ -78,6 +83,43 @@ trait PersistenceDAO extends LazyLogging {
       )
       logger.debug(s"Wrote document $document to ${written.toUri}")
     }
+
+  /**
+    * Return the GCS bucket metadata files for `index` upserted on or
+    * after `upsertId`.  Decode the files from JSON as `ClioDocument`s
+    * sorted by `upsertId`.
+    *
+    * @param upsertId of the last known upserted document
+    * @param index  to query against.
+    * @param ec is the implicit ExecutionContext
+    * @tparam D is a ClioDocument type with a JSON Decoder
+    * @return a Future Seq of ClioDocument
+    */
+  def getAllSince[D <: ClioDocument: Decoder](
+    upsertId: UUID,
+    index: ElasticsearchIndex[D]
+  )(implicit ec: ExecutionContext): Future[Seq[D]] =
+    Future {
+      val rootDirectory = rootPath.resolve(index.currentPersistenceDir)
+      val files = Try(Files.newDirectoryStream(rootDirectory))
```

Comment:
Is the ordering assumption of the output seq relying on `Files.newDirectoryStream` to return the names in order? If so, could you add a comment?

Reply:
According to the javadoc, `The elements returned by the iterator are in no specific order.`

Reply:
Yep. I sort them now.